static/app/components/searchQueryBuilder/tokens/filterKeyListBox/index.tsx

```diff
@@ -125,7 +210,7 @@ export function FilterKeyListBox<T extends SelectOptionOrSectionWithKey<string>>
               ref={listBoxRef}
               listState={state}
               hasSearch={false}
-              hiddenOptions={hiddenOptions}
+              hiddenOptions={modifiedHiddenOptions}
```

Comment:
Is this so they don't appear twice? TBH I don't think it'd be a bad thing if they did, kinda like sticky notes in a phone book or something, I wouldn't expect them to be removed from the bigger list.

Reply:
I'll actually say I would find it odd if they were removed from the greater list, since if someone were to ignore suggestions, they'd scroll down to the lists and find their options missing.

Reply:
I'll clarify this in the code with a comment since it is confusing, but this `hiddenOptions` simply prevents the recent filters items from showing in the `<ListBox />` component. They have keys that look like `__recent_filter__<name>` so the keys don't conflict with the normal items. We use `hiddenOptions` because the whole component shares a collection state so it's difficult to just remove the items from the list here.

Reply:
I see, so were hiding rendering them twice, not omitting them from the list itself, makes sense. Thanks for clarifying!